qemu/tests/cfg/sgx_cpu.cfg

```diff
@@ -0,0 +1,33 @@
+- sgx_cpu:
+    type = sgx_cpu
+    only Linux
+    kill_vm = yes
+    login_timeout = 240
+    image_snapshot = yes
+    use_mem = no
+    mem_devs = "epc0 epc1"
+    backend_mem_epc0 = "memory-backend-epc"
+    size_mem_epc0 = 64M
+    prealloc_epc0 = on
+    backend_mem_epc1 = "memory-backend-epc"
+    size_mem_epc1 = 28M
+    prealloc_epc1 = on
+    vm_sgx_epc_devs = sgx0 sgx1
+    vm_sgx_epc_memdev_sgx0 = epc0
+    vm_sgx_epc_node_sgx0 = 0
+    vm_sgx_epc_memdev_sgx1 = epc1
+    vm_sgx_epc_node_sgx1 = 0
+    RHEL.9:
+        repo_install_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm"
+    RHEL.8:
+        repo_install_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
+    cpuid_pkg = "cpuid"
+    check_cpuid_sgx_cmd = "cpuid -1 |grep -i 'replace_string.*= true'"
```

Comment:
options:
check_cpuid_sgx_cmd = "cpuid -1 |grep -i '%s.*= true'"
params['check_cpuid_sgx_cmd'] % i

or if you want a meaningful name, try
check_cpuid_sgx_cmd = "cpuid -1 |grep -i '{meaningful_name}.*= true'"
params['check_cpuid_sgx_cmd'].format(meaningful_name=xxx)

just fyi, yours is OK for me